src/actions/index.js

```diff
@@ -0,0 +1,59 @@
+import rootReducer from "../reducers";
+
+export function loadMyMovieList() {
+ return function (dispatch) {
+   dispatch({
+     type: 'LOAD_MY_MOVIE_LIST'
+   });
+   fetch('/movies')
+   .then( (response) => {
+     return response.json();
+   }).then((movies) => {
+     dispatch(myMovieListLoaded(movies));
+   });
+ };
+}
+
+export function myMovieListLoaded(movies){
+  return {
+    type:'MY_MOVIE_LIST_LOADED',
+    value:movies
+  }
+}
+
+export function loadSearch(searchTerm) {
+ return function (dispatch) {
+   dispatch({
+     type: 'LOAD_SEARCH'
+   });
+   fetch("https://api.themoviedb.org/3/movie/550?api_key=c3b94590ff0e2dd196b19855e2")
```

Comment:
I don't think this api key is valid. Double check the URL from the README. You left some things out of it